src/openrct2/platform/Platform.Posix.cpp

```diff
@@ -59,6 +59,31 @@ namespace Platform
         return std::string(value, colon);
     }
 
+    std::string GetOsName()
+    {
+        struct utsname buffer;
+
+        errno = 0;
+        if (uname(&buffer) < 0)
+        {
+            log_error("Couldn't get OS");
+            return std::string
+            {
+            }
+        }
+
+        std::string output = buffer.sysname + " " + buffer.release + " " + buffer.version;
+
+        if (buffer.machine == "x86_64")
```

Comment:
I was reviewing this on mobile and missed the fact this is done in`Posix.cpp`, not `Linux.cpp`, so that probably addresses my comment about Mac os.

Still, you have to change this to `strcmp` and add at least `arm64`

Reply:
Just removed the check and directly added in the machine type